templates/github/.github/workflows/fips.yml.j2

```diff
@@ -0,0 +1,64 @@
+{% include 'header.j2' %}
+---
+name: Pulp FIPS Nightly CI
+on:
+  schedule:
+    # * is a special character in YAML so you have to quote this string
+    # runs at 4:00 UTC daily
+    - cron: '00 4 * * *'
+
+jobs:
+  vagrant:
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - test_type: pulp3-source-centos8-fips
+          - test_type: pulp3-source-centos7-fips
+    steps:
+      - uses: actions/checkout@v2.3.1
+        with:
+          submodules: true
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
+      - name: Ansible
+        run: |
+          pip3 install -U pip
+          pip3 install "ansible>2.9,<3"
+          ansible --version
+      - name: Pulp Installer
+        run: |
+          cd ..
+          git clone --depth=1 https://github.com/pulp/pulp_installer
+      - name: Install Vagrant
+        shell: bash
+        run: |
+          cd ../pulp_installer/
+          git submodule update --init
+          {%- if plugin_name in ['pulp_file', 'pulpcore'] %}
+          sed -i "/{{ plugin_name }}/d" .github/workflows/scripts/install.sh
+          {%- endif %}
+          .github/workflows/scripts/install.sh {{ "${{ matrix.test_type }}" }}
+      - name: Run Vagrant
+        shell: bash
+        run: |
+          cd ../pulp_installer
+          {%- if plugin_name not in ['pulp_file', 'pulpcore'] %}
+          sed -i "s/pulp-file/{{ plugin_dash }}/g" example.dev-config.yml
+          sed -i "s/pulp_file/{{ plugin_name }}/g" example.dev-config.yml
```

Comment:
```suggestion
          sed -e "s/pulp-file/{{ plugin_dash }}/g" -e "s/pulp_file/{{ plugin_name }}/g" example.dev-config.yml > local.dev-config.yml
```

Reply:
You can perform multiple scripts in one `sed` call. Also i prefer not to change git controlled files (but that is a matter of taste).

Reply:
Nice! I didn't know about that